python/heterocl/tvm/runtime.py

```diff
@@ -2,8 +2,19 @@
 import os, subprocess, time, re, glob
 from ..report import parse_xml
 from ..devices import Project
+import glob
+
 debug = True
 
+def locate_xilinx_vitis():
+    vitis_path = "/opt/xilinx/"
+    env_cmd = ""
+    for directory in os.listdir(vitis_path):
+        if "_vitis_" in directory or "-vitis-" in directory:
+            env_cmd = f"source {vitis_path}/{directory}/Vivado/2019.2/settings64.sh; source /opt/xilinx/xrt/setup.sh; "
```

Comment:
It is not wise to use a fixed version (2019.2) here. Could you figure out other ways to improve the portability? The code should not only work on our server, but also on other machines.

Reply:
Do you think it is safe to assume that `vitis_path` is set appropriately?

Reply:
For a temporary solution, you can suppose the Vivado toolset is installed in `/opt/xilinx/`. If we cannot find that folder or run the setting script, we should throw out an error to suggest users check their installation path in`/opt/xilinx`.

Reply:
For the second question, yes.